services/pg-backup/docker-compose.yml

```diff
@@ -1,14 +1,20 @@
 version: '3.7'
 services:
-  dbbackup:
-    image: justb4/pgbackup:10
+  SERVICE_NAME_PARAMETRIZED:
+    image: kartoza/pg-backup:14-3.3@sha256:4d1867707b1da879324199fb3c0d542122113bfc50cd2418764efb8f4309d6c1
     volumes:
-    - /var/run/docker.sock:/var/run/docker.sock
-    - /tank:/backup
+      - /tank/${MACHINE_FQDN}:/backups
     environment:
-    - PGB_SCHEDULE=0 * * * * # Every hours
+      - DUMPPREFIX=PG_${MACHINE_FQDN}
+      - DUMP_ARGS="--format=c"
```

Comment:
What does this mean, what is the resulting format? Let's try to stick with vanilla .sql files if possible is my suggestion

Reply:
SQL dump give something 7 times bigger (11M with the custom format, around 77M with vanilla). You will have more details here : https://www.postgresql.org/docs/current/app-pgdump.html - look for format. I will add a comment

Reply:
OK I want to give it one last try to persuade you:
In the past, for the pg14 upgrade, I had to manually mess with the `.sql` files since there was a custom function defined for some relations in the DB that was wrong, but pg11 was fine with it. PG14 however would not accept it. From this experience, I personally am happy to accept 7 times bigger backup sizes, since anyway this is hosted as a service for us by O+P. So I dont see a problem with large space needs. We should delete old backups older than >1 year anyway every now and then ;)

Your thoughts?

Reply:
I agree, let's go for it.

Reply:
I added a policy to delete them after 1 year.